pkg/sink/sink_test.go

```diff
@@ -433,28 +421,267 @@ func Test_HandleEvent(t *testing.T) {
 	ts := httptest.NewServer(http.HandlerFunc(r.HandleEvent))
 	defer ts.Close()
 
-	numRequests := 0
 	restClient.Client = fakerestclient.CreateHTTPClient(func(request *http.Request) (*http.Response, error) {
-		numRequests += 1
-		if diff := cmp.Diff(wantPipelineResourceURLPath, request.URL.Path); diff != "" {
-			t.Errorf("Diff request uri: -want +got: %s", diff)
-		}
-		body, err := ioutil.ReadAll(request.Body)
-		if err != nil {
-			return nil, err
-		}
-		if diff := cmp.Diff(string(wantPipelineResourceBytes), string(body)); diff != "" {
-			t.Errorf("Diff request body: -want +got: %s", diff)
-		}
```

Comment:
@khrm 
Sorry, this is pretty late, but why was this block of code removed? Now there's nothing checking that the correct payload was sent to the kubernetes api.

Reply:
It wasn't thread safe. So I have removed it. And forgot to add a thread safe version. Sending a pr soon.
I had added response to check atleast that it reached till the end.

Reply:
Thanks! https://github.com/tektoncd/triggers/pull/121